File: pythonCursoEmVideo/Ex024.py
# solicitando o nome completo do usuario
nome = str(input('Digite o seu nome completo: ')).strip().title() # tirando espaços indesejados e deixando primeiras letras das palavras em Caixa alta

print('Seu nome tem Oliveira?', 'Oliveira' in nome)# procurando a palavra oliveira dentro da string
print(nome)

File: JavaScript-Devmedia/Nova pasta/Ex024.js
let usuarios = [
    {id: 1, nome: "Jussiêr", idade: 25},
    {id: 2, nome: "Miguel", idade: 15},
    {id: 3, nome: "Oliveira", idade: 10}
];

for(let usuario of usuarios){
    let id = usuario.id;
    let nome = usuario.nome;
    let idade = usuario.idade;
    console.log(id);
    console.log(nome);
    console.log(idade);
}